### ISSUE TITLE
Spork fails when launchy is used

### ISSUE THREAD

**Issue by NickClark** [sporkrb/spork#107]
I am trying to use cucumber with launchy, which works, untill I try to use it with spork. Then I get a **#<TypeError: incompatible marshal file format (can't be read)** error. Heres my gist with the complete error: https://gist.github.com/956091

**Update:** I have also created an  [example project](https://github.com/NickClark/Launchy-Test) to illustrate the problem. Although, I'm not certain if this is a spork, launchy, or growl problem now.

**Comment by ltw** [comment]
This issue has been raised on the launchy issue tracker, and closed as probably not a launchy issue. https://github.com/copiousfreetime/launchy/issues/13

**Comment by 23inhouse** [comment]
Did you guys get any resolution on this?

**Comment by NickClark** [comment]
No. Its a pain. If I want to run that command, I have to run cucumber by itself. Instead of shutting spork down though, I usually just open a different terminal window and run cucumber with a different port. Otherwise it will find the spork instance and run through that anyways. Like this: `cucumber --port 0 path/to/my/feature.feature`

**Comment by 23tux** [comment]
Although this issue is opened for 2 years now, I also get this error when running spork with evergreen + launchy. Is there any fix available?
